applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -1003,16 +1003,23 @@ static void ad_parse(struct net_buf_simple *p_ad, const bt_addr_le_t *addr)
 static void on_device_found(const bt_addr_le_t *addr, int8_t rssi, uint8_t type,
 			    struct net_buf_simple *p_ad)
 {
-	/* Direct advertising has no payload, so no need to parse */
-	if (type == BT_GAP_ADV_TYPE_ADV_DIRECT_IND) {
+	switch(type){
+	case BT_GAP_ADV_TYPE_ADV_DIRECT_IND:
+		/* Direct advertising has no payload, so no need to parse */
 		if (bonded_num) {
 			bt_foreach_bond(BT_ID_DEFAULT, bond_connect, (void *)addr);
 		}
-		return;
-	} else if ((type == BT_GAP_ADV_TYPE_ADV_IND || type == BT_GAP_ADV_TYPE_EXT_ADV) &&
-		   (bonded_num < CONFIG_BT_MAX_PAIRED)) {
+		break;
+	case BT_GAP_ADV_TYPE_ADV_IND:
+	case BT_GAP_ADV_TYPE_EXT_ADV:
+	case BT_GAP_ADV_TYPE_SCAN_RSP:
```

Comment:
This SCAN_RSP type did not previously end up in this if. Can you comment?

Reply:
I'll move the SCAN_RSP addition to a later commit then.

Reply:
The SCAN_RSP

Reply:
The SCAN_RSP was actually done in a later commit. After the squash, this detail was gone. :)